packages/dm-core-plugins/package.json

```diff
@@ -24,16 +25,16 @@
     "@types/react-dom": "^18.2.7",
     "@types/react-router-dom": "^5.3.3",
     "@types/styled-components": "^5.1.26",
+    "@typescript-eslint/eslint-plugin": "^6.7.3",
+    "@typescript-eslint/parser": "^6.7.3",
+    "eslint-plugin-react": "^7.33.2",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
     "react-router-dom": ">=5.1.2",
     "shx": " 0.3.4",
     "ts-jest": "^29.1.0",
     "ts-loader": "^9.3.1",
-    "typescript": "^5.1.6",
-    "eslint-plugin-react": "^7.33.2",
-    "@typescript-eslint/eslint-plugin": "^6.7.3",
-    "@typescript-eslint/parser": "^6.7.3"
+    "typescript": "^5.1.6"
```

Comment:
switcheroo? personal preference, but I would probably not include this in the PR since it is not related.